packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/models/country_code.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:widget_toolkit/models.dart';
+
+/// Country code model
+class CountryCodeModel extends PickerItemModel {
+  CountryCodeModel({
+    required this.code,
+    required this.name,
+  });
+
+  /// Name of the country
+  final String name;
+
+  /// Code of the country
+  final String code;
+
+  CountryCodeModel.empty()
+      : code = '',
+        name = '';
+
+  factory CountryCodeModel.fromJson(Map<String, dynamic> json) {
```

Comment:
Can't we use json_seriazible instead of doing it by hand ?

Reply:
This was generated by Copilot. Fixed now.